src/Chapter1.hs

```diff
@@ -556,8 +569,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
-
+isVowel :: Char -> Bool
+isVowel c
+    | c `elem` "aeiou" = True
```

Comment:
Nice work on using infix form of calling functions! We don't explain this syntax in our course to make it short. But well done :+1: 

As `elem` returns `Bool` itself, you can just return the result of this function:

```haskell
isVowel c = c `elem` "aeiou"
```

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎